### DISCUSSION TITLE
Dependency Limit Reached

### DISCUSSION THREAD

**Discussion by Gurshaan7** [aptos-labs/aptos-developer-discussions#318] [Questions]
### Describe your question in detail.

While deploying a contract , I received Dependency limit reached error , Whats the limit here? And how its calculated , let say If I call coin::deposit function two times will that be counted as one or two.

### Which operating system are you using?

Linux (Ubuntu, Fedora, Windows WSL, etc.)

### Which SDK or tool are you using? (if any)

Aptos CLI

**Comment by 0x-j** [comment]
I've never run into this error before, can you share your source code? or the move.toml file
